CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to Chipy.org
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the Chicago Python User Group website, which are hosted in the [Chicago Python User Group Organization](https://github.com/chicagopython). These are guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request (PR).
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Chipy Code of Conduct](https://www.chipy.org/pages/conduct/). By participating, you are expected to uphold this code. Please report unacceptable behavior [here](https://www.chipy.org/contact/).
+
+## How Can I Contribute?
+
+Unsure where to begin contributing? You can start by looking through issues tagged `difficulty: easy` on the [issues page](https://github.com/chicagopython/chipy.org/issues).
+
+### Local Development
+
+Chipy.org can be deployed and viewed locally. For instructions on how to do this, see [README.md](README.md).
+
+### Pull Requests
+
+Please follow these steps to make your first PR!
+
+1. Fork this repository by clicking the fork button at the top of the page. This will create an instance of the entire repository in your own account.
+2. Clone the repository fork to your machine to work with it locally. On the GitHub for your fork, click the clone button and copy the link. Then, run the following commands to clone the repo and move into it:
+    ```
+    $ git clone https://github.com/YOUR_USERNAME_HERE/chipy.org
+
+    $ cd chipy.org
+    ```
+3. Create a new branch with a short, descriptive name of the changes you'll make:
+    ```
+    $ git checkout -b BRANCH_NAME
+    ```
+4. Make changes to the branch to fix the issue you selected
```

Comment:
Line 4, add period to end of sentence.

Reply:
@sophm256 line 4 is a blank line, did you mean somewhere else?

Reply:
Sorry, I meant please add period to the end of line 34 (which was step 4).

Reply:
Gotcha! Fix will be pushed in next commit